src/main/java/frc/robot/subsystems/drivetrain/SwerveDrive.java

```diff
@@ -24,11 +24,10 @@ public class SwerveDrive extends SubsystemBase {
             SwerveConstants.wheelPositions[3]
     );
     private final SwerveDriveOdometry odometry;
-    private final SwerveDriveInputsAutoLogged loggerInputs = new SwerveDriveInputsAutoLogged();
-    private final SwerveModuleState[] currentModuleStates = new SwerveModuleState[4];
-    private final SwerveModuleState[] desiredModuleStates = new SwerveModuleState[4];
+    private SwerveDriveInputsAutoLogged loggerInputs = new SwerveDriveInputsAutoLogged();
```

Comment:
Looks like `SwerveDriveInputsAutoLogged` is missing from the repository. Did you push it?